src/entry-point-client/PageList.tsx

```diff
@@ -55,6 +56,12 @@ const PageList: PageInfo[] = [
     isPrivate: true,
     contents: <ModelInspectionPage />,
   },
+  {
+    path: "/impact",
+    title: getPageTitle("Response impact"),
```

Comment:
Nit: Response Impact

I know the model inspection page doesn't have the full titular capitalization but that's also a typo